app/src/main/java/com/infomaniak/mail/ui/main/thread/ThreadFragment.kt

```diff
@@ -578,6 +578,10 @@ class ThreadFragment : Fragment() {
         }
     }
 
+    private fun observeMessagesIsCollapsable() = with(threadViewModel) {
+        isCollapsable.observe(viewLifecycleOwner, threadAdapter::messagesCollapseStateChange)
+    }
```

Comment:
We usually bother with `with(threadViewModel)` when it's accessed more than once in the method but not otherwise